plugins/tracker-resources/src/components/projects/Projects.svelte

```diff
@@ -0,0 +1,84 @@
+<!--
+// Copyright © 2022 Hardcore Engineering Inc.
+// 
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+
+<script lang="ts">
+  import { FindOptions, Ref, SortingOrder } from '@anticrm/core'
+  import { Button, IconAdd, Label, showPopup } from '@anticrm/ui'
+  import { Table } from '@anticrm/view-resources'
+  import { Project, Team } from '@anticrm/tracker'
+  import NewProject from './NewProject.svelte'
+  import plugin from '../../plugin'
+
+  export let space: Ref<Team>
+
+  const options: FindOptions<Project> = {
```

Comment:
Please use lookup for assignee to obtain it with server request.

Reply:
Done